[2024-01-08T12:49:40.263+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: our_first_dag_v3.second_task manual__2024-01-08T12:49:35.948544+00:00 [queued]>
[2024-01-08T12:49:40.282+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: our_first_dag_v3.second_task manual__2024-01-08T12:49:35.948544+00:00 [queued]>
[2024-01-08T12:49:40.283+0000] {taskinstance.py:2171} INFO - Starting attempt 1 of 6
[2024-01-08T12:49:40.312+0000] {taskinstance.py:2192} INFO - Executing <Task(BashOperator): second_task> on 2024-01-08 12:49:35.948544+00:00
[2024-01-08T12:49:40.325+0000] {standard_task_runner.py:60} INFO - Started process 4151 to run task
[2024-01-08T12:49:40.330+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v3', 'second_task', 'manual__2024-01-08T12:49:35.948544+00:00', '--job-id', '10', '--raw', '--subdir', 'DAGS_FOLDER/our_first_dag.py', '--cfg-path', '/tmp/tmpwbn0ul8n']
[2024-01-08T12:49:40.336+0000] {standard_task_runner.py:88} INFO - Job 10: Subtask second_task
[2024-01-08T12:49:40.425+0000] {task_command.py:423} INFO - Running <TaskInstance: our_first_dag_v3.second_task manual__2024-01-08T12:49:35.948544+00:00 [running]> on host d3b0837e9e03
[2024-01-08T12:49:40.572+0000] {taskinstance.py:2481} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='arif' AIRFLOW_CTX_DAG_ID='our_first_dag_v3' AIRFLOW_CTX_TASK_ID='second_task' AIRFLOW_CTX_EXECUTION_DATE='2024-01-08T12:49:35.948544+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-01-08T12:49:35.948544+00:00'
[2024-01-08T12:49:40.577+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-01-08T12:49:40.579+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo hey, I am task2 and will be running after the task1!']
[2024-01-08T12:49:40.606+0000] {subprocess.py:86} INFO - Output:
[2024-01-08T12:49:40.611+0000] {subprocess.py:93} INFO - hey, I am task2 and will be running after the task1!
[2024-01-08T12:49:40.612+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-01-08T12:49:40.661+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v3, task_id=second_task, execution_date=20240108T124935, start_date=20240108T124940, end_date=20240108T124940
[2024-01-08T12:49:40.702+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-01-08T12:49:40.737+0000] {taskinstance.py:3281} INFO - 0 downstream tasks scheduled from follow-on schedule check
